qa/README.md

```diff
@@ -0,0 +1,121 @@
+# QA Checklist
+
+Name: <Name>
+
+Date: <yyyy-mm-dd>
+
+Renku helm chart version: <0.25.0>
+
+
+## Projects
+
+- [ ] Create a new project on Renku (UI)
+  <details>
+    <summary>Details</summary>
+
+    1. Create a new project in the website
+    2. Ensure the KG integration is active
+    3. Ensure a session can be successfully started on the project
+    4. Ensure said sesssion can be accessed
+  </details>
+
+- [ ] Delete an existing project (UI)
+  <details>
+    <summary>Details</summary>
+
+    1. Create a new project in the website
+    2. Wait fo the image build and KG integration to complete
+    3. Delete the project
+    4. Ensure the project is truly gone
+  </details>
+
+- [ ] Fork a project (UI)
+  <details>
+    <summary>Details</summary>
+
+    1. Create a new public project in the website
+    2. Wait fo the image build and KG integration to complete
+    3. Log in as a different user
+    4. Fork the project
+    5. Ensure that KG integration succeeds in the fork
+    6. Ensure that sessions can be launched in the fork
+  </details>
+
+
+## Datasets
+
+- [ ] Create a dataset from the UI
+  <details>
+    <summary>Details</summary>
+
+    1. Create a new Renku project in the website
+    2. Download a dataset from https://zenodo.org/
+    3. Upload the data and create the dataset from the UI
```

Comment:
If we are downloading data from zenodo, why not test the import functionality? 

If testing dataset creation with uploaded data, just find some cat photos online and upload them

Reply:
I was thinking that the data found on Zenodo is more representative of a research/data science dataset. A few cat photos are small and easy to handle usually. I think striving to handle any Zenodo file is a good thing. I imagine once in a while someone will pick a file that is too large or that causes problems. With cat photos it is more likely to be trivial. But yeah cat photos is fine too. Dogs too 😸

Reply:
yeah maybe - but then if we're just going to zenodo to get some data (and not using the zenodo import functionality) then we should just put some large-ish files somewhere for people to grab. Finding something usable/useful in zenodo can be hit-or-miss in my experience.